modules/bling/README.md

```diff
@@ -19,5 +19,5 @@ install:
     # - devpod # https://devpod.sh/ as an rpm
     # - gnome-vrr # enables gnome-vrr for your image 
     # - container-tools # installs container-related tools onto /usr/bin: kind, kubectx, docker-compose and kubens 
-
+    # - laptop # installs TLP and configures your system for laptop usage # depends on "files: - laptop: /usr" to work!
```

Comment:
> # depends on "files: - laptop: /usr" to work!

what does this mean?

Reply:
quite literally that! the user should add the "laptop" usr tree to make this install script work properly since it adds the tlp config. but yeah, maybe this should be a bit reworded

Reply:
Is there a way to just copy that folder from the script? If so, that might be the best solution